lib/include/openamp/rpmsg.h

```diff
@@ -99,6 +101,7 @@ struct rpmsg_device_ops {
 	int (*send_offchannel_nocopy)(struct rpmsg_device *rdev,
 				      uint32_t src, uint32_t dst,
 				       const void *data, int len);
+	int (*release_tx_buffer)(struct rpmsg_device *rdev, void *rxbuf);
```

Comment:
rxbuf->txbuf

Reply:
but not change

Reply:
The [compare ](https://github.com/OpenAMP/open-amp/compare/806f192983475d06cb64930e832b2cdc8d4853ea..148ae1d1da65d48415eb485fccb0c413c0daf9d6) button associated to my force-update shows the  delta
Please, could check it again?

Reply:
It look good now.

Reply:
uhm, I still see `rxbuf` here.
![Screenshot from 2022-07-13 11-56-43](https://user-images.githubusercontent.com/357758/178706614-6d1f15be-94e6-4b39-a3aa-c365ae5a9e69.png)

Reply:
My apologize i was not fix the good commit, done should be ok